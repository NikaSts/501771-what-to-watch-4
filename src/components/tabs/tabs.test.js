import React from 'react';
import renderer from 'react-test-renderer';
import Tabs from './tabs';

const tabs = [`Overview`, `Reviews`, `Details`];
const activeItem = tabs[0];
const activeMovie = {
  id: 1,
  title: `Active Movie`,
  runTime: 1,
  genre: ``,
  releaseDate: 1,
  poster: ``,
  previewImage: ``,
  backgroundImage: ``,
  backgroundColor: ``,
  ratingScore: 1,
  ratingLevel: ``,
  ratingCount: 1,
  description: ``,
  director: ``,
  stars: [``],
  preview: ``,
  video: ``
};

it(`Tabs should render correctly`, () => {
  const tree = renderer
    .create(
        <Tabs
          activeMovie={activeMovie}
          activeItem={activeItem}
          onItemClick={() => {}}
        />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
