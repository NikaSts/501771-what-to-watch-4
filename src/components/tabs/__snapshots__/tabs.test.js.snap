// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tabs should render correctly 1`] = `
<div
  className="movie-card__desc"
>
  <nav
    className="movie-nav movie-card__nav"
  >
    <ul
      className="movie-nav__list"
    >
      <li
        className="movie-nav__item movie-nav__item--active"
      >
        <a
          className="movie-nav__link"
          href="#"
          onClick={[Function]}
        >
          Overview
        </a>
      </li>
      <li
        className="movie-nav__item"
      >
        <a
          className="movie-nav__link"
          href="#"
          onClick={[Function]}
        >
          Reviews
        </a>
      </li>
      <li
        className="movie-nav__item"
      >
        <a
          className="movie-nav__link"
          href="#"
          onClick={[Function]}
        >
          Details
        </a>
      </li>
    </ul>
  </nav>
  <div
    className="movie-rating"
  >
    <div
      className="movie-rating__score"
    >
      8,9
    </div>
    <p
      className="movie-rating__meta"
    >
      <span
        className="movie-rating__level"
      >
        Very good
      </span>
      <span
        className="movie-rating__count"
      >
        240
         ratings
      </span>
    </p>
  </div>
  <div
    className="movie-card__text"
  >
    <p>
      Quentin Tarantino's Once Upon a Time... in Hollywood visits 1969 Los Angeles,
  where everything is changing, as TV star Rick Dalton (Leonardo DiCaprio) and his longtime
  stunt double Cliff Booth (Brad Pitt) make their way around an industry they hardly recognize
  anymore. The ninth film from the writer-director features a large ensemble cast and multiple
  storylines in a tribute to the final moments of Hollywood's golden age.
    </p>
    <p
      className="movie-card__director"
    >
      <strong>
        Director: 
        Quentin Tarantino
      </strong>
    </p>
    <p
      className="movie-card__starring"
    >
      <strong>
        Starring: 
        Leonardo DiCaprio, Brad Pitt, Margot Robbie
         and other
      </strong>
    </p>
  </div>
</div>
`;
